conductr_cli/conduct_info_inspect.py

```diff
@@ -156,6 +158,7 @@ def display_bundle_executions(bundle):
             for row in rows:
                 log.screen('{endpoint: <{endpoint_width}}{padding}'
                            '{host: <{host_width}}{padding}'
+                           '{pid: <{pid_width}}{padding}'
```

Comment:
pid is a number and therefore should be right aligned, e.g. as `>{bind_port_width}`